pixart.js

```diff
@@ -1 +1,36 @@
+var button = document.querySelector("#set-color");
+var colorText = document.querySelector("#color-field");
+var bigSquare = document.querySelector(".brush");
+var color = "#1B4370";
+var pixels = document.querySelector(".pixels");
 
+playPixart();
+
+function playPixart() {
+  button.addEventListener("click", setColor); //This also works when "enter" is pressed on the keyboard. Why? Should I create a separate event to ensure this event happens for keying enter on any computer?
+  addSquares();
+  pixels.addEventListener("mouseover", clickSquare);
+}
```

Comment:
Your code works, and you can certainly solve your problem by adding the event listener on the entire pixels div, but maybe it would be easier to add the event listener on each individual div with a class of square? Just a thought